backend/backend_test.go

```diff
@@ -52,6 +56,11 @@ func testBackend(b backend.Backend, t *testing.T) {
 			OK(t, err)
 			Assert(t, rd != nil, "Get() returned nil")
 
+			// try to read it out again
+			r, err := b.GetReader(tpe, test.id, 0, uint(len(test.data)))
+			OK(t, err)
+			Assert(t, r != nil, "GetReader() returned nil")
```

Comment:
is that all we care about? that it's not `nil`? don't we also want to assert that you can actually use this reader to read things?